ztp/gitops-subscriptions/argocd/CoreReduction.md

```diff
@@ -0,0 +1,37 @@
+# Core Reduction with Workload Partitioning and Performance Profile
+
+Maximum number of cores can be allocated for customer workload only by pinning host and cluster workloads to a small number of cores. This is known as "core reduction" and be achieved by correctly configuring Workload Partitioning and Performance Profile.
```

Comment:
Same note here, technically the act of segregating platform workloads from customer workloads on different CPUSets is workload partitioning not core reduction. So something like this might be a good summary of what we want to cover in this section, thoughts?
```
Workload partitioning is the feature that allows the platform to dedicate specific CPUSets to customer workloads and platform workloads. This feature **must** be activated at install time of a cluster, once active that cluster is locked into functioning with this feature on.
```

Reply:
used your suggestion